### PR TITLE
QA-4256 Change GridGain version for ignite-binding module

### PR DESCRIPTION
**Pull request by nsivkov**
https://ggsystems.atlassian.net/browse/QA-4256

### CODE REVIEW

**Review by iartiukhov — APPROVED** [review]
LGTM
@Berkof Could you also take a look?

File: ignite/pom.xml [hunk 1]
```diff
@@ -36,25 +36,33 @@ LICENSE file.
...
+    </repository>
+  </repositories>
+
   <dependencies>
     <dependency>
-      <groupId>org.apache.ignite</groupId>
+      <groupId>org.gridgain</groupId>
       <artifactId>ignite-core</artifactId>
-      <version>${ignite.version}</version>
+      <version>${gridgain.version}</version>
```
Comment (iartiukhov) [inline ignite/pom.xml:53]: Should not we use `<scope>provided</scope>` here for all `org.gridgain` artifacts?

Comment (nsivkov) [reply ignite/pom.xml:53]: Probably. I'm not sure. `ignite3-binding` module hasn't `provided` scope for AI3 as well.

Comment (nsivkov) [reply ignite/pom.xml:53]: Updated and tested with `provided` scope. Works well with [poc-tester branch qa-4256](https://github.com/ggprivate/poc-tester/tree/qa-4256).

**Review by iartiukhov — APPROVED** [review]

**Review by ahitrin — APPROVED** [review]

**Pull request merged**